e2e/playwright/share-link.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { expect, test } from '@e2e/playwright/zoo-test'
+
+const isWindows =
+  navigator.platform === 'Windows' || navigator.platform === 'Win32'
+test.describe('Share link tests', () => {
+  ;[
+    {
+      codeLength: 1000,
+      showsErrorOnWindows: false,
+    },
+    {
+      codeLength: 2000,
+      showsErrorOnWindows: true,
+    },
+  ].forEach(({ codeLength, showsErrorOnWindows }) => {
+    test(`Open in desktop app with ${codeLength}-long code ${isWindows && showsErrorOnWindows ? 'shows error' : "doesn't show error"}`, async ({
```

Comment:
Clever. I guess this will show up as two different tests in the report. Probably fine. 😄

Reply:
Haha I hope that won't break your stuff!